static/index.html

```diff
@@ -2,11 +2,12 @@
 <html<% if (APPCACHE) { %> manifest="manifest.appcache"<% } %>>
 
     <head>
+        <base href="/android_asset/www/">
```

Comment:
I'm not really sure how else do to this.

There are a lot of paths that need to be relative (fonts, images, etc.). 

I'm sure this would need to be re-written for ffos and ios.